### ISSUE TITLE
Golang insecure cookie rule false positive case

### ISSUE THREAD

**Issue by elsapet** [Bearer/bearer-rules#351]
## Description & Reproduction

The following code should not trigger the `go_lang_cookie_missing_http_only` rule

```go
func Ok(w http.ResponseWriter, name, value string) {
	cookie := http.Cookie{
		Name:     name,
		Value:    value,
		HttpOnly: true,
		Secure:   true,
	}
	http.SetCookie(w, &cookie)
}
```

## Expected Behavior

The code should not trigger the `go_lang_cookie_missing_http_only` rule

## Actual Behavior

The rule is triggered

## Your Environment

- Operating System and version:
- Output of 'bearer version':
```
bearer version: 0.00.0
sha: xxx
```
